src/main/groovy/org/mockito/release/internal/comparison/DownloadPreviousReleaseArtifactsTask.java

```diff
@@ -0,0 +1,82 @@
+package org.mockito.release.internal.comparison;
+
+import org.gradle.api.DefaultTask;
+import org.gradle.api.GradleException;
+import org.gradle.api.logging.Logger;
+import org.gradle.api.logging.Logging;
+import org.gradle.api.tasks.*;
+import org.mockito.release.notes.util.IOUtil;
+
+import java.io.File;
+
+/**
+ * Downloads artifacts from last release and stores them to local files for further comparison
+ * Currently it downloads .pom and -sources.jar
+ */
+public class DownloadPreviousReleaseArtifactsTask extends DefaultTask {
+
+    private static final Logger LOG = Logging.getLogger(DownloadPreviousReleaseArtifactsTask.class);
+
+    @Input
+    private String previousVersionPomUrl;
+    @Input
+    private String previousVersionSourcesJarUrl;
+
+    @OutputFile
+    private File previousVersionPomLocalFile;
+    @OutputFile
+    private File previousVersionSourcesJarLocalFile;
+
+
+    @TaskAction public void downloadPreviousReleases(){
+        if(previousVersionPomUrl == null || previousVersionSourcesJarUrl == null){
```

Comment:
This is automatically checked by Gradle because you are using @Input annotation. You can just remove the entire 'if' block.